.gitignore

```diff
@@ -1,3 +1,5 @@
-target/*
-project/target/*
+/target/
+/project/target/
 
+# IntelliJ IDEA
+/.idea/
```

Comment:
Not connected to Akka 2.0.2 - just for your consideration.

I find anchoring rules to the project root / often good, and would recomment the above two-line sbt ignore patterns. The ones you have could kick in if one has target folders within the project. Matter of taste.

And I used IntelliJ IDEA, so added the exclude for the IDE files.